app/components/windows/ChildWindow.vue.ts

```diff
@@ -75,6 +71,7 @@ export default class ChildWindow extends Vue {
     clearTimeout(this.refreshingTimeout);
     this.refreshingTimeout = window.setTimeout(() => {
       this.components.push({ name: options.componentName, isShown: true });
+      electron.remote.getCurrentWindow().setTitle(this.options.title);
```

Comment:
Does it work for windows with `preservePrevWindow=true` like media gallery?

Reply:
good catch, will try to get it working on any shown window